samples/subsys/logging/syst/sample.yaml

```diff
@@ -1,11 +1,23 @@
 sample:
   name: syst sample
 tests:
-  sample.logger.syst:
+  sample.logger.syst.v1:
     platform_allow: mps2_an385 sam_e70_xplained
     tags: logging
     harness: console
     harness_config:
       type: one_line
       regex:
         - "SYS-T RAW DATA: "
+    extra_configs:
+      - CONFIG_LOG2_MODE_DEFERRED=n
```

Comment:
This should not need to be disabled. v2 logging is not enabled by default.

Reply:
Removed the disabling for v1 logging in sample.yaml.